Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN set -eux; \
   fi && \
   yum clean all && rm -rf /var/cache/yum
 
-FROM rockylinux:8.6.20227707@sha256:afd392a691df0475390df77cb5486f226bc2b4cbaf87c41785115b9237f3203f AS rockylinux8
+FROM rockylinux:8.6.20227707 AS rockylinux8
```

Comment:
I remember that generally we prefer using the `@sha256` notation with a mutable tag because dependabot just check for freshness of tags - not semantic or worse a timestamp like in this case

Reply:
I changed it because it didn't work. the tag was updated upstream from `afd392a691df0475390df77cb5486f226bc2b4cbaf87c41785115b9237f3203f`to `b820932e70edf6559d190a1f2e7b19f080ff45a61fc485bb4aad2d1b38b64e5d` and dependabot didn't pick it up while it did pick up 8.5 to 8/6 update